.travis.yml

```diff
@@ -33,7 +33,7 @@ env:
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
         - EVENT_TYPE='pull_request push'
-        - PIP_DEPENDENCIES='scipy matplotlib ads synphot https://github.com/astropy/astroquery/archive/master.zip pytest-astropy codecov'
+        - PIP_DEPENDENCIES='numpy scipy matplotlib ads synphot https://github.com/astropy/astroquery/archive/master.zip pytest-astropy sphinx-astropy'
```

Comment:
Are changes in `.travis.yml` needed for this PR? Or are they an artifact of the rebase?

Reply:
Oh. Artifact! Definitely. Will remove this in another commit. :D

Reply:
It looks good now :p